internal/messages/errormessages.go

```diff
@@ -33,6 +33,7 @@ const (
 	Email                             = "email"
 	Firstname                         = "firstname"
 	Lastname                          = "lastname"
+	Locale                            = "locale"
```

Comment:
improving the proximity factor :)

Reply:
Not really... I added a "new" constant... but it could not be built because it was already present so I removed one but not removed mine... I'll revert this change